langgraph_prototype/local_visualization.py

```
@@ -0,0 +1,342 @@
+    print("LangSmithアクセス問題の代替可視化ソリューション")
+    print()
+    
+    visualizer = LocalLangGraphVisualizer()
+    
+    # 1. 構造図の生成
+    print("1. 📊 構造図生成")
+    visualizer.create_mermaid_html()
+    
+    try:
+        visualizer.generate_graphviz_diagram()
+    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)